ddtrace/contrib/pymemcache/client.py

```diff
@@ -14,18 +18,36 @@
 from ddtrace import Pin
 from ddtrace.compat import reraise
 from ddtrace.ext import memcached as memcachedx
+from ddtrace.ext import net
+
+log = logging.getLogger(__name__)
+
 
 # keep a reference to the original unpatched clients
 _Client = Client
 
 
+DEFAULT_SERVICE = os.environ.get("DATADOG_SERVICE_NAME") or memcachedx.SERVICE
```

Comment:
not sure we should use `DATADOG_SERVICE_NAME`. `pymemcache` is not an entry point (in general) and if someone sets the value as their application name (i.e. `intake-api`), the latency and the breakdown for `pymemcache` will not be computed (you don't see the req/s in the service page etc...).

I think it should be just `memcachedx.SERVICE`. People then will use the `Pin` object to change the default value. You can attach the Pin to `pymemcache` package (we support that) so when a new wrapper is created you will inherits the `Pin` (see the comment below).